http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -622,4 +658,12 @@ private JsonWebEncryption encryptRequest(JsonWebSignature signedRequest) throws
             return jsonEncryption;
         }
     }
+
+    private String generateSessionId() {
+        SecureRandom random = new SecureRandom();
+        byte[] nonceData = new byte[NONCE_SIZE];
+        random.nextBytes(nonceData);
+        return String.valueOf(ByteIterator.ofBytes(nonceData)
```

Comment:
I think this can be `return ByteIterator.ofBytes(nonceData).base64Encode().drainToString();`

Reply:
done